lib/marc_links.rb

```diff
@@ -18,7 +18,7 @@ def as_h
       {
         version: '0.1',
 
-        html: ["<a title='#{link_title}' href='#{link_field["u"]}'>#{link_text}</a>", "#{'(source: Casalini)' if link_is_casalini?}", (" <span class='additional-link-text'>#{additional_text}</span>" if additional_text)].compact.join(' '),
+        html: [%Q(<a title="#{link_title}" href="#{link_field['u']}">#{link_text}</a>), "#{'(source: Casalini)' if link_is_casalini?}", (%Q( <span class="additional-link-text">#{additional_text}</span>) if additional_text)].compact.join(' '),
```

Comment:
Should we just go ahead and treat all those values as untrusted and HTML escape them? Or, maybe compose the link downstream in Searchworks in the first place?

Reply:
What would it take to compose it in SearchWorks? that sounds like a better approach, potentially

Reply:
I agree that SearchWorks would be a better place for generating the html links. The linking test coverage in SW seems pretty good, but it will definitely take some time to untangle things on the SW side.

Reply:
I pushed up a change to this PR to HTML escape the strings that get embedded in these chunks of HTML. Moving the HTML link generation into SearchWorks makes a lot of sense to me, but it'll take some time a consideration about how to approach it. Will create a separate issue for that.